WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -455,6 +458,15 @@ private async Task HandleCoinJoinFinalizationAsync(CoinJoinTracker finishedCoinJ
 			wallet.LogError($"{nameof(CoinJoinClient)} failed with exception: '{e}'");
 		}
 
+		// If any coins were marked for banning, store them to file
+		if (finishedCoinJoin.PrisonedCoins.Any())
+		{
+			foreach (var info in finishedCoinJoin.PrisonedCoins)
+			{
+				CoinPrison.Ban(info.Coin, info.BanUntilUtc);
+			}
```

Comment:
We could have a `Ban` function that takes List<CoinBanned> as a parameter.

Just a suggestion, feel free to ignore.

Reply:
If not here, then inside the version that takes in a list - we need a foreach. 
I'd rather keep the function handy for single coin banning.